containers/utils/download-zowe-build.sh

```diff
@@ -16,10 +16,14 @@ cd /root/zowe
 
 if [ "$ZOWE_BUILD" = "latest" ]; then
   echo "Downloading most recent release ..."
+  apt-get update && apt-get install -y wget --no-install-recommends
   wget -q "https://zowe.jfrog.io/zowe/libs-release-local/org/zowe/[RELEASE]/zowe-[RELEASE].pax" -O "$ZOWE_TARGET_NAME"
+  apt-get purge -y wget
```

Comment:
we remove wget that we use during build time, but maybe its something we should keep in the image for use by plugins?